finance.md

```diff
@@ -0,0 +1,69 @@
+# Financial Accounts and Practices
+
+_Note: Note all of the following have been implemented yet, the Finance WG is working on it_
+
+This document outlines finance and accounting practices of Hypha Worker Co-operative, including:
+
+- Listing of accounts at our financial institutions
+- Process for invoicing clients and accepted methods of bill payments
+- Process for payroll and expense reimbursement to Members
+- Tools and procedures for bookkeeping, accounting, and tax remittances
+
+## Organizational structure
+
+Hypha Worker Co-operative is a worker co-operative incorporated in Ontario, Canada. The business has seven Founding Members, four of which are First Directors with signing authority. As a non-share-capital worker co-operative, we are a non-profit organization, and every Member holds both a Worker and an Owner relationship with the business.
```

Comment:
```suggestion
Hypha Worker Co-operative is a worker co-operative incorporated in Ontario, Canada. The business has seven Founding Members, of which three of the four First Directors have with signing authority. As a non-share-capital worker co-operative, we are a non-profit organization, and every Member holds both a Worker and an Owner relationship with the business.
```

Reply:
what is a "worker" relationship?

Could you say: every Member is both an employee and co-owner of the business